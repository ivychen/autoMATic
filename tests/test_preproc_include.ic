#include "stdlib_test.ic"

int main() {
	test();
	print(number());
	return 0;
}
