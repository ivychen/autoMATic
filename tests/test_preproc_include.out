imported test stdlib
100
